demo-app/src/main/java/io/opentelemetry/android/demo/MainActivity.kt

```diff
@@ -73,9 +74,13 @@ class MainActivity : ComponentActivity() {
                             painterResource(id = R.drawable.otel_icon),
                         )
                         val context = LocalContext.current
-                        OpenStoreButton(text = "Click to begin", onClick = {
+                        OpenStoreButton(text = "Go shopping", onClick = {
                             context.startActivity(Intent(this@MainActivity, AstronomyShopActivity::class.java))
                         })
+                        OpenStoreButton(text = "Learn more", onClick = {
```

Comment:
It's clear now that this button isn't just for opening the store....maybe it can be renamed to something more generic, like `LauncherButton` or something.